src/js/letters/utils/helpers.js

```diff
@@ -1,8 +1,9 @@
 /* eslint-disable camelcase */
-
+import React from 'react';
 import merge from 'lodash/fp/merge';
 
 import environment from '../../common/helpers/environment';
+import { formatDateShort } from '../../post-911-gib-status/utils/helpers';
```

Comment:
Yes definitely, I thought about it and then was too lazy to do it in the moment. Will update!